webrender/src/frame.rs

```diff
@@ -319,6 +319,31 @@ impl Frame {
                 continue;
             }
 
+            let mut delta:Point2D<f32> = match scroll_location {
+                ScrollLocation::Delta(delta) => delta,
+                ScrollLocation::Start => {
+                    if layer.scrolling.offset.y.round() == 0.0 {
+                        // Nothing to do.
+                        return false;
```

Comment:
from my own discussions with @mrobinson with regards to https://github.com/servo/webrender/pull/533 I am pretty much certain that, as @glennw suggested in https://github.com/servo/webrender/pull/540#discussion_r87093181, those `return` statements should be turned into `continue`, because the loop would still need to scroll any other layer with the same `scroll_root_id`. 

So instead of returning a boolean, you could do something like `scrolled_a_layer = scrolled_a_layer || your_boolean`, followed by `continue;`